extensions/3DTILES_tile_metadata/README.md

```diff
@@ -0,0 +1,161 @@
+# 3DTILES_tile_metadata
```

Comment:
In the next iteration we should include support for arbitrary sized arrays instead of just `VEC2`, `VEC3`, `VEC4`, etc. Same goes for glTF [`EXT_3dtiles_feature_metadata`](https://github.com/CesiumGS/glTF/pull/1). It seems like eventually everything's going to conform to the same schema.